x-pack/filebeat/module/cisco/ftd/_meta/fields.yml

```diff
@@ -0,0 +1,107 @@
+- name: ftd
+  type: group
+  description: >
+    Fields for Cisco Firepower Threat Defense Firewall.
+  fields:
+  - name: message_id
+    type: keyword
+    description: >
+      The Cisco FTD message identifier.
+
+  - name: suffix
+    type: keyword
+    example: session
+    description: >
+      Optional suffix after %FTD identifier.
+
+  - name: source_interface
+    type: keyword
+    description: >
+      Source interface for the flow or event.
+
+  - name: destination_interface
+    type: keyword
+    description: >
+      Destination interface for the flow or event.
+
+  - name: list_id
+    type: keyword
+    description: >
+      Name of the Access Control List that matched this event.
+
+  - name: source_username
+    type: keyword
+    description: >
+      Name of the user that is the source for this event.
+
+  - name: destination_username
+    type: keyword
+    description: >
+      Name of the user that is the destination for this event.
+
+  - name: mapped_source_ip
+    type: ip
+    description: >
+      The translated source IP address.
+
+  - name: mapped_source_port
+    type: long
+    description: >
+      The translated source port.
+
+  - name: mapped_destination_ip
+    type: ip
+    description: >
+      The translated destination IP address.
+
+  - name: mapped_destination_port
+    type: long
+    description: >
+      The translated destination port.
+
+  - name: threat_level
+    type: keyword
+    description: >
+      Threat level for malware / botnet traffic. One of very-low, low,
+      moderate, high or very-high.
+
+  - name: threat_category
+    type: keyword
+    description: >
+      Category for the malware / botnet traffic. For example: virus, botnet,
+      trojan, etc.
+
+  - name: connection_id
+    type: keyword
+    description: >
+      Unique identifier for a flow.
+
+  - name: icmp_type
+    type: short
+    description: >
+      ICMP type.
+
+  - name: icmp_code
+    type: short
+    description: >
+      ICMP code.
+
+  - name: security
+    type: object
+    description:
+      Raw fields for Security Events.
+
+
+  # TODO: These DNS fields must be moved outside cisco.ftd once dns support
+  #       is added to ECS.
+  - name: dns.question.name
```

Comment:
This is waiting for https://github.com/elastic/ecs/pull/438 to be merged. Otherwise I'll keep it and add a description to the fields.

Reply:
It has been merged now. A new release of ECS will be made on Aug 22 then we can update the ECS fields.yml we have in Beats.